client/view/index/Index.html

```diff
@@ -135,5 +209,44 @@ <h2>Try an example deck</h2>
 	components: {
 		SpreadsheetInput,
 	},
-}
+	methods: {
+		prepareDeleteSeenWorkbook: function(element) {
+			var sure = document.querySelector(".are-you-sure-window");
+			sure.style.display = "block";
+			sure.style.opacity = '1';
```

Comment:
With Svelte, you shouldn't ever have to manipulate the DOM manually like this.  One way to have these kind of dynamic styles on an element is by putting a data attribute on the element in the template:

```html
<div class="are-you-sure-window" data-is-visible={showDialog}>
	Are you sure?
</div>
```
and then add a style that will apply when appropriate:
```css
.are-you-sure-window[data-is-visible=true] {
	display: block;
	opacity: 1;
}
```